### DISCUSSION TITLE
Many Countries do not appear in the 2FA SMS options (Need Offical Response)

### DISCUSSION THREAD

**Discussion by raza2022** [community/community#78943] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

**Please don't close it with `copied text `from another old response we need an official response on behalf of millions of users around the world whose countries missing from that `2FA SMS list`**

Yesterday I received the message
`Your account meets this criteria, and you will need to enroll in 2FA within 45 days, by January 26th, 2024 at 00:00 (UTC). After this date, your access to GitHub.com will be limited until you enroll in 2FA. Enrolling is easy, and we support several options, starting with TOTP apps and text messages (SMS) and then adding on passkeys and the GitHub Mobile app.`

When I tried to enable SMS/Text messages, I was surprised to see that PAKISTAN+92 was not in the list, I researched more and saw many countries missing from that list, how on earth can an enterprise do this to push your time users without complete mandatory steps,
There should be an official response/message from GitHub, because as far as I know my country is in 4th position in freelancer/tech-based economies
I do not wanna seem bitter, but sadly, now I remember my friends who suggested moving from GitHub after the Microsoft acquisition

**Comment by Albert7866** [comment]
Users globally, including those in Pakistan, are experiencing issues with GitHub's 2FA SMS enrollment due to missing [country ](http://housemoversinabudhabi.com)codes. An official response from GitHub addressing this omission is needed to ensure comprehensive access for all users.
